pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go

```diff
@@ -33,13 +33,16 @@ import (
 	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
 	cm "github.com/rook/rook/pkg/operator/ceph/nvmeof_recoverer/clustermanager"
 	"github.com/rook/rook/pkg/operator/ceph/reporting"
```

Comment:
add watch predicate for OSD pod fault events --> add watch predicate to detect  OSD pod fault events